packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue

```diff
@@ -1,27 +1,43 @@
 <template>
 	<aside class="overlay-container">
 		<section>
-			<header ref="header">
-				<div>
-					<slot name="header" />
-				</div>
-				<Button
-					icon="pi pi-times"
-					text
-					rounded
-					aria-label="Close"
-					@click="emit('on-close-clicked')"
-				/>
-			</header>
-			<main><slot /></main>
+			<tera-drilldown-header
+				:active-index="selectedViewIndex"
+				:views="views"
+				:tooltip="tooltip"
+				@tab-change="handleTabChange"
+				@close="emit('on-close-clicked')"
+				>{{ props.title }}</tera-drilldown-header
+			>
+			<main>
+				<template v-for="(view, index) in views">
+					<slot v-if="selectedViewIndex === index" :name="view" />
```

Comment:
I'm not sure we should switch between tabs like this, this is cause for re-render at every switch (see the spinner in the video).

Reply:
Im slight changing this so that `display: none` is applied instead so that it does not render the tab each time:

```
<main>
    <div v-for="(view, index) in views" class="slot-container" :class="{'hide-slot': hideSlot(index)}">
	    <slot :name="view" />
    </div>
</main>
```

Reply:
Maybe

```vue
<section v-for="(view, index) in views" class="slot-container" :class="{'hide-slot': hideSlot(index)}">
  <slot :name="view" />
</section>
```